front50-api/src/main/java/com/netflix/spinnaker/front50/api/validator/PipelineValidator.java

```diff
@@ -14,15 +14,22 @@
  * limitations under the License.
  */
 
-package com.netflix.spinnaker.front50.validator;
+package com.netflix.spinnaker.front50.api.validator;
 
 import com.netflix.spinnaker.front50.model.pipeline.Pipeline;
```

Comment:
I'm wondering what Spinnaker people will think of including a front50 model in the api package? I haven't seen models included in other api packages for other projects other than Fiat.

Reply:
Mentioned this in the same comment about `front50-core` - unfortunately, pulling in `Pipeline` implicitly pulls in a bunch of other stuff you don't want.